README.rst

```diff
@@ -22,43 +23,102 @@ Query Parameters:
 * ?[columnheader1]=[value1]&[columnheader3]=[value4]  (returns results that have both value1 and value2)
 
 
+Requirements
+------------
+
+In order to use autoapi, you'll need:
+
+* Python 3.5
+* npm
```

Comment:
As far as I can tell, these are the dependencies? Let me know if they need changing... I'm also not sure what particular version of npm is needed--it seems like it doesn't much matter since we're just using npm as a package manager, and not using nodejs at all...